frontend/src/metabase/admin/tools/components/Logs/Logs.tsx

```diff
@@ -53,8 +53,8 @@ const LogsBase = ({
   const hasAnyLogs = logs.length > 0;
   const { scrollRef, onScroll, refollow } = useTailLogs(filteredLogs);
   const logText = useMemo(
-    () => filteredLogs.map(formatLog).join("\n"),
-    [filteredLogs],
+    () => filteredLogs.map(formatLog(process, processUUIDs)).join("\n"),
+    [filteredLogs, process, processUUIDs],
```

Comment:
given that this is going to be called a lot, you should call the generator just once and then reuse the output
```tsx
() => {
    const formatter = formatLog(process, processUUIds);
    return filteredLogs.map(formatter).join("\n")
 }
```